BugReport/Repository.cs

```diff
@@ -205,16 +213,28 @@ public void LoadIssues()
         }).Wait();
     }
 
+    public void LoadSubscribedIssues()
+    {
+        GitHubClient client = CreateGitHubClient();
+
+        RepositoryIssueRequest issueRequest = new RepositoryIssueRequest
+        {
+            State = ItemStateFilter.All,
+            Filter = IssueFilter.Subscribed
+        };

```

Comment:
I am curious if this is going to hit API limit on my account on CoreFX ...

Reply:
There's no way to filter for "unsubscribed", so the options are either to check individual issues or to get bulk subscribed. I'm hoping get bulk is faster.